packages/atomic/template/stencil.config.ts

```diff
@@ -5,13 +5,27 @@ import html from 'rollup-plugin-html';
 // https://stenciljs.com/docs/config
 
 export const config: Config = {
+  namespace: '{{project}}',
   globalStyle: 'src/style/index.css',
   taskQueue: 'async',
   outputTargets: [
     {
       type: 'www',
       serviceWorker: null, // disable service workers
-      copy: [{src: 'pages', keepDirStructure: false}],
+      copy: [
+        {src: 'pages', keepDirStructure: false},
+        {
+          src: '../node_modules/@coveo/atomic/dist/atomic',
+          dest: 'atomic',
+          keepDirStructure: false,
+        },
+      ],
+    },
+    {
+      type: 'dist',
+    },
+    {
+      type: 'dist-custom-elements',
```

Comment:
Adding these because I had some feedback that some users don't know how to bundle their custom components. Adding these targets (https://stenciljs.com/docs/distribution) will help them bundle their components and deploy them. Some additional docs will have to be written for that.